### PR TITLE
blank pictures, hide related buttons, chunk up

### PR DESCRIPTION
**Pull request by fiji-flo**
- blank pictures when switching profiles
- hide list/grid button when only 0/1 directs
- load deps async

### CODE REVIEW

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 1]
```diff
@@ -419,7 +420,7 @@ export default {
     initialValues: Object,
   },
   components: {
-    Combobox,
+    Combobox: () => import('@/components/ui/Combobox.vue'),
```
Comment (jens1o) [inline src/components/profile/edit/EditPersonalInfo.vue]: why is this loaded lazily, although (at least afaic) it is loaded everytime?

File: src/components/profile/edit/EditPictureModal.vue [hunk 2]
```diff
@@ -106,7 +104,7 @@ export default {
     staffInformation: Object,
   },
   components: {
-    Crop,
+    Crop: () => import('./Cropper.vue'),
```
Comment (jens1o) [inline src/components/profile/edit/EditPictureModal.vue]: same here?

File: src/components/profile/edit/EditPersonalInfo.vue (hunk 1 above)
Comment (fiji-flo) [reply src/components/profile/edit/EditPersonalInfo.vue]: I'm starting to chunk up the dependencies. This produces multiple chunks in a `npm run build` (check it out). See https://vuejs.org/v2/guide/components-dynamic-async.html
The goal is to lazy load them at some point. So theses components are only loaded when a user edits their profile.

Comment (jens1o) [reply src/components/profile/edit/EditPersonalInfo.vue]: Ah, I see. Weird semantic, though

**Pull request merged**